### ISSUE TITLE
Chaos on home page

### ISSUE THREAD

**Issue by reyesvicente** [KRTirtho/spotube#1296]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The playlists and everything showing in the page moves unexpectedly

### Expected Behavior

The playlists and other stuff not moving

### Steps to reproduce

1. Open the app
2. Do nothing

### Operating System

Kali Linux

### Spotube version

v3.4.1

### Installation source

GitHub Releases (Binary), Flathub (Flatpak)

### Additional information

[Screencast from 2024-03-06 18-20-13.webm](https://github.com/KRTirtho/spotube/assets/45283313/2bcc5e6f-938b-40ee-b498-308a16aee349)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by olivier2** [comment]
Duplicate of #1142 ?  I think it's a cache related issue (the data needs a refresh) combined with an expired Spotify token that happens after a long idle time. I've seen it on Windows and Mac too.

**Comment by reyesvicente** [comment]
> Duplicate of #1142 ? I think it's a cache related issue (the data needs a refresh) combined with an expired Spotify token that happens after a long idle time. I've seen it on Windows and Mac too.

Right. A duplicate of [#1142](https://github.com/KRTirtho/spotube/issues/1142) How do we stop this from happening? Token isn't expired. After I reopen it, it's back to normal.

**Comment by KRTirtho** [comment]
Duplicate of #1142
